web/extensions/core/editAttention.js

```diff
@@ -58,7 +69,8 @@ name:id,
 
         function editAttention(event) {
             const inputField = event.composedPath()[0];
-            const delta = 0.025;
+            let delta = localStorage.getItem("Comfy.Settings.Comfy.EditAttention.Delta") || "0.1"
```

Comment:
Is there a reason why the 0.1 default value has to be in 2 different places?

Reply:
I guess not. Fixed.